src/Protector.php

```diff
@@ -341,11 +334,10 @@ public function getGitBranch(): string
     /**
      * Generates an SQL dump from the current app database and returns the path to the file.
      *
-     * @param string $destinationFilePath
      * @param array $options
-     * @return bool
+     * @return string|null
      */
-    protected function generateDump(string $destinationFilePath, array $options = []): bool
+    protected function generateDump(array $options = []): ?string
```

Comment:
is this a good place to check if the file is empty before we leave this method?

Reply:
Done.